### PR TITLE
Christmas trees will give out anything gifts on christmas day

### PR DESCRIPTION
**Pull request by Simplehorror**
# Document the changes in your pull request
Have a little HOLIDAY spirit

# Changelog

:cl:  
tweak: Christmas trees will give out anything gifts on christmas day
/:cl:

### CODE REVIEW

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -70,6 +70,11 @@
 	. = ..()
 	if(!took_presents)
 		took_presents = list()
+	
+	var/MM = text2num(time2text(world.timeofday, "MM"))
+	var/DD = text2num(time2text(world.timeofday, "DD")) 
+	if (DD == 25 && MM == DECEMBER)
```
Comment (Bibby0110) [inline code/game/objects/structures/flora.dm:76]:
```suggestion
	if (DD == 24 && MM == DECEMBER)
```

Comment (Simplehorror) [reply code/game/objects/structures/flora.dm:76]: is it fucking zero aligned????

**Comment by JamieD1** [comment]
Anythings are quite strong, will look into

**Comment by Simplehorror** [comment]
> Anythings are quite strong, will look into

don't be a grinch

**Comment by MajManatee** [comment]
Its one day a year jamie, have some holiday spirit, sheesh..

**Comment by JamieD1** [comment]
> Its one day a year jamie, have some holiday spirit, sheesh..

Yeah but its a big fucking admin issue you know, "SOMEONE KILLED ME WITH A METEOR LAUNCHER WAHHH"

**Comment by Simplehorror** [comment]
reopen this

**Pull request closed without merging**